app/Http/Controllers/GameController.php

```diff
@@ -60,7 +64,18 @@ public function getBorderMarkers(Game $game)
 
     public function getNotifications(Game $game)
     {
-        return $game->notifications()->get();
+        if (Request::get('all') === 'true')
```

Comment:
Beetje raar dat er voor is gekozen om `true` als string-waarde te sturen en niet als boolean.

Reply:
Omdat je hem uit de URL krijgt. Dat is standaard tekst, en casting werkt hier niet voor.